bundles/model/org.eclipse.smarthome.model.core/src/main/java/org/eclipse/smarthome/model/core/internal/folder/FolderObserver.java

```diff
@@ -238,6 +238,9 @@ private static void checkFile(final ModelRepository modelRepo, final File file,
                         if (parsers.contains(getExtension(file.getName()))) {
                             try (FileInputStream inputStream = FileUtils.openInputStream(file)) {
                                 modelRepo.addOrRefreshModel(file.getName(), inputStream);
+                            } catch (Exception e) {
```

Comment:
with this, you are catching away all exceptions from [below](https://github.com/eclipse/smarthome/pull/3277/files#diff-06b195932ce2dfc1602326f23f1a79d2R252) and shield them with a very generic error log. Maybe it would make sense to keep the original catch block, reduce the severity to warn and avoid printing the stacktrace there. Otherwise I'd suggest to simply add a catch of IOException to the existing catch block and handle only those like that.